src/Javis.jl

```diff
@@ -491,28 +500,49 @@ function perform_transformation(trans::InternalRotation)
     rotate(trans.angle)
 end
 
-pos(p::Point) = p
-pos(t::Transformation) = t.p
+"""
+    val(s::Symbol)
+
+Get access to the value that got saved in `s` by a previous action.
+If you want to access a position or angle check out [`pos`](@ref) and [`angle`](@ref).
 
-function pos(s::Symbol)
+# Returns
+- `Any`: the value stored by a previous action.
+"""
+function val(s::Symbol) 
     defs = CURRENT_VIDEO[1].defs
     if haskey(defs, s)
-        pos(defs[s])
+        return defs[s]
     else
         error("The symbol $s is not defined.")
     end
 end
 
+pos(p::Point) = p
+pos(t::Transformation) = t.p
+
+"""
+    pos(s::Symbol)
+
+Get access to the value that got saved in `s` by a previous action.
+If you want to access a position or angle check out [`pos`](@ref) and [`angle`](@ref).
+
+# Returns
+- `Point`: the point stored by a previous action.
+"""
+pos(s::Symbol) = pos(val(s))
+
 angle(t::Transformation) = t.angle
 
-function angle(s::Symbol)
-    defs = CURRENT_VIDEO[1].defs
-    if haskey(defs, s)
-        angle(defs[s])
-    else
-        error("The symbol $s is not defined.")
-    end
-end
+"""
+    angle(s::Symbol)
+
+Get access to the angle that got saved in `s` by a previous action.
+
+# Returns
+- `Float64`: the angle stored by a previous action i.e via `return Transformation(p, angle)`
+"""
+angle(s::Symbol) = angle(val(s))
```

Comment:
Merge conflict with #66 based on naming -- just a heads-up for you to be aware of.

Reply:
Yeah it was a hard one but I think I managed it.